atlassian-terraform/sumologic.auto.tfvars

```diff
@@ -0,0 +1,20 @@
+# Sumo Logic - Atlassian Terraform
+# Configure Sumo Logic credentials and App installation options.
+
+# Sumo Logic
+sumo_access_id          = "suZGHyWOCpae0J" # https://help.sumologic.com/Manage/Security/Access-Keys
+sumo_access_key         = "Cp4PFmcX95yMFQOQctToPjGwoYpQcy8YGUN4ZsbLUqFsm6ZiSW8zVrznOhZX8MGh"
+deployment              = "us1"                                    # https://help.sumologic.com/APIs/General-API-Information/Sumo-Logic-Endpoints-and-Firewall-Security
+sumo_api_endpoint       = "https://nite-api.sumologic.net/api/v1/" # https://help.sumologic.com/APIs/General-API-Information/Sumo-Logic-Endpoints-and-Firewall-Security
```

Comment:
Don't expose internal endpoints.

Reply:
Wrong file was uploaded. Correct file with placeholders has been uploaded and all the credentials are revoked.